dist/msi/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ else()
     set(TR_OPENSSL_SSL_NAME "ssleay32.dll")
 endif()
 
-set(ICONS_DIR "${CMAKE_SOURCE_DIR}/qt/icons/hicolor")
+set(ICONS_DIR "${CMAKE_SOURCE_DIR}/qt/icons/Win11")
```

Comment:
This change has nothing to do with the PR title. Still, if you want to change this line and don't want to open a separate PR (or move it to another PR where it'll be more at home), I'd suggest removing the line instead because the variable it declares (`ICONS_DIR`) is not used; IIRC it was used before when this script was combining a set of PNG files into an ICO file, but the latter is now committed to the repo, no combining is taking place, and this is a simple leftover.

Since each PR should be isolated, this change also doesn't make a lot of sense even if the variable was used, because there is no "qt/icons/Win11" directory in current `main`, so it should be "qt/icons/Faenza" instead. And even then, that subdirectory is not the place to store our application icon.

Reply:
Thanks @mikedld i'll make all of those amendments and also remove the /dist/msi changes, it's better in as part of #4376.